test/e2e/test_complete_workflow.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
My earlier question about where this went is answered.

One could argue that this test belongs in the media-insights repo since this workflow is part of that application.  Would we add new application workflows here as well?

Reply:
No. This test isn't testing the specific workflow that the media insights app uses. Instead, the goal for this test is to eventually test a workflow with every operator in the operator library and every supported file format.

Right now it just so happens that the two workflows look very similar since we're only trying to get to an "mvp" coverage state. That'll change as we add support for more file formats, validate the input/output of the operators, and have full operator library coverage.